packages/sheet2code-ui/src/render-page/client-entry.js

```diff
@@ -0,0 +1,26 @@
+import { ThemeProvider } from '@material-ui/core/styles'
+import * as buildConsts from '../constants/build'
+import App from '../components/app'
+import React from 'react'
+import ReactDOM from 'react-dom'
+import set from 'lodash/set'
+import theme from '../constants/theme'
+
+function renderAppTo(renderTo, appProps) {
+  const targetElement =
+    typeof renderTo === 'string' ? document.querySelector(renderTo) : renderTo
+  ReactDOM.hydrate(
+    <ThemeProvider theme={theme}>
+      <App {...appProps} />
+    </ThemeProvider>,
+    targetElement
+  )
+}
+
+if (typeof window !== 'undefined') {
+  set(
+    window,
+    [buildConsts.namespace, buildConsts.pkgName, 'renderAppTo'],
+    renderAppTo
+  )
+}
```

Comment:
What does this file do?
I guess this file is used by clients to build the webpack bundles?

Reply:
To use webpack to build the bundle mentioned [here](https://github.com/twreporter/orangutan-monorepo/pull/5/files/cb16b2dffc934ccc4c1cfcc56d0b145430c74c27#r398353455)

Reply:
I've not added the webpack cofig and make script to build the bundle now. I'll add them after rebasing the `@twreporter/oragutan` merged master